.gitignore

```diff
@@ -1,3 +1,4 @@
+my_log_file.log
```

Comment:
What if you add a date to the naming pattern and it will be a different file?
It's best to add the log folder here so that any files there don't end up in the repository.